apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -1,90 +1,37 @@
-import { fieldType, text as textOption } from '../../constant/constant';
+import { text as textOption } from '../../constant/constant';
 import { ModalComponent } from '../../../../components/ModalComponent/ModalComponent';
 import { ValuesetLibrary } from '../../pages/ValuesetLibrary/ValuesetLibrary';
+import { Controller, useForm } from 'react-hook-form';
+import { Input } from '../../../../components/FormInputs/Input';
+import { SelectInput } from '../../../../components/FormInputs/SelectInput';
+import { maxLengthRule } from '../../../../validation/entry';
 
 export const CreateQuestion = ({ modalRef, question }: any) => {
-    const init = {
-        label: '',
-        description: '',
-        type: UpdateQuestionRequest.type.TEXT,
-        subgroup: '',
-        uniqueId: '',
-        uniqueName: '',
-        defaultLabelInReport: '',
-        defaultRdbTableName: '',
-        rdbColumnName: '',
-        includedInMessage: false,
-        messageVariableId: '',
-        labelInMessage: '',
-        codeSystem: '',
-        hl7DataType: ',',
-        HL7Segment: ',',
-        requiredInMessage: false,
-        dataMartColumnName: '',
-        adminComments: '',
-        codeSet: 'PHIN',
-        relatedUnits: 'No',
-        futureDates: 'No',
-        displayType: '',
-        reportLabel: '',
-        defaultValue: 'test@gmai.com',
-        fieldLength: '50',
-        mask: 'TXT',
-        fieldSize: '50',
-        tooltip: '',
-        displayControl: 0,
-        minValue: 0,
-        maxValue: 0,
-        allowFutureDates: false,
-        dateFormat: ''
-    };
-    const validation = {
-        label: { required: true, error: false, fb: false },
-        description: { required: false, error: false, fb: false },
-        subgroup: { required: true, error: false, fb: false },
-        uniqueId: { required: true, error: false, fb: false },
-        uniqueName: { required: true, error: false, fb: false },
-        defaultLabelInReport: { required: true, error: false, fb: false },
-        defaultRdbTableName: { required: true, error: false, fb: false },
-        rdbColumnName: { required: true, error: false, fb: false },
-        dataMartColumnName: { required: false, error: false, fb: false },
-        messageVariableId: { required: true, error: false, fb: false },
-        labelInMessage: { required: true, error: false, fb: false },
-        codeSystem: { required: true, error: false, fb: false },
-        hl7DataType: { required: true, error: false, fb: false },
-        tooltip: { required: true, error: false, fb: false },
-        displayType: { required: true, error: false, fb: false },
-        fieldLength: { required: true, error: false, fb: false },
-        minValue: { required: true, error: false, fb: false },
-        maxValue: { required: true, error: false, fb: false },
-        dateFormat: { required: true, error: false, fb: false }
-    };
-
+    const { handleSubmit, reset, control } = useForm<any, any>();
```

Comment:
Can we have separate `useForm` invocations for the various question types? (Date, Text, Numeric, Coded). This allows us to set the defaults and leverage typing instead of `<any, any>`

Declaring the forms in this manner allows us to more easily specify the default selections for different quest types. Example

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/a351e62a-858c-4f11-a9d2-0c44081e7f34)

We could have different sub components for the different question types.

So based on the `selectedFieldType`, different input components would be displayed.

something like:
```typescript
const textQuestionForm = useForm<CreateTextQuestionRequest>();
const numericQuestionForm = useForm<CreateNumericQuestionRequest>( { defaultValues: { 
    minValue: 0,
    maxValue: 50
    }
} );
 {selectedFieldType === 'TEXT' ? <CreateTextQuestionFields  form={textQuestionForm} /> : null}
 {selectedFieldType === 'NUMERIC' ? <CreateNumericQuestionFields  form={numericQuestionForm} /> : null}
```